src/app/public/src/modules/grid/column.component.ts

```diff
@@ -1,3 +1,7 @@
+/**
+ * Deprecated, update and use the @skyux/fluid-grid unless major bugs are discovered before full deprecation in v3.
```

Comment:
Not a big deal, but could use the JSDoc `@deprecated` annotation.
http://usejsdoc.org/tags-deprecated.html